starlite/handlers/http.py

```diff
@@ -258,13 +283,16 @@ def validate_handler_function(self) -> None:
         if "data" in signature.parameters and "GET" in self.http_methods:
             raise ImproperlyConfiguredException("'data' kwarg is unsupported for 'GET' request handlers")
 
-    def _get_response_from_data(
+    def get_response_from_data(
```

Comment:
Given that `HTTPRouteHandler` is part of our public api, I'd prefer to leave this as non-public implementation detail until there was a good reason to expose them.

Same would apply to `resolve_after_response()` and `_process_after_request_hook()`. 

Just my opinion, but is there a reason that a user should ever need to call these directly?

Reply:
It's probably better to keep the public API as simple as possible.  I can't think of a reason for a user to call these at the moment, but I'm good either way we land on this one.

Reply:
fixed